<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Mapa_Ruta">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapa Ruta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AQUI VA TU DESCR......</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Recibir_pedidos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Recibir Pedidos y Generar Rutas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mapeo">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapeo</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_Pedidos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>671</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>165</number>
    </attribute>
    <column>
     <property name="text">
      <string>Pedidos</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Garrafones</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ubicacion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Beneficio</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
